system-int-test/test/cmr/system_int_test/search/scrolling_search_test.clj

```diff
@@ -0,0 +1,128 @@
+        coll2 (data2-core/ingest-umm-spec-collection "PROV1" 
+                                                     (data-umm-c/collection {:EntryTitle "E2"
+                                                                             :ShortName "S2"
+                                                                             :Version "V2"}))
+        coll1-cid (get-in coll1 [:concept-id])
+        coll2-cid (get-in coll2 [:concept-id])
+        gran1 (data2-core/ingest "PROV1" 
+                                 (data2-granule/granule-with-umm-spec-collection coll1 
+                                                                                 coll1-cid 
+                                                                                 {:granule-ur "Granule1"}))
+        gran2 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll1 
+                                                                          coll1-cid 
+                                                                          {:granule-ur "Granule2"}))
+        gran3 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll1 
+                                                                          coll1-cid 
+                                                                          {:granule-ur "Granule3"}))
+        gran4 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll2 
+                                                                          coll2-cid 
+                                                                          {:granule-ur "Granule4"}))
+        gran5 (data2-core/ingest "PROV1" 
+                          (data2-granule/granule-with-umm-spec-collection coll2 
+                                                                          coll2-cid 
+                                                                          {:granule-ur "Granule5"}))
+        all-grans [gran1 gran2 gran3 gran4 gran5]]
+    (index/wait-until-indexed)
+               
+    (testing "Scrolling with page size"
+      (let [{:keys [hits scroll-id] :as result} (search/find-refs :granule {:provider "PROV1" :scroll true :page-size 2})]
+        (testing "First call returns scroll-id and hits count with page-size results"
+          (is (= (count all-grans) hits))
+          (is (not (nil? scroll-id)))
+          (is (data2-core/refs-match? [gran1 gran2] result)))
+        
+        (testing "Subsequent searches gets page-size results"
+          (let [result (search/find-refs :granule {:scroll true :scroll-id scroll-id})]
+            (is (= (count all-grans) hits))
+            (is (data2-core/refs-match? [gran3 gran4] result))))
+
+        (testing "Remaining results returned on last search"
+          (let [result (search/find-refs :granule {:scroll true :scroll-id scroll-id})]
```

Comment:
I remember you saying scroll-id was 720 characters - are we really expecting people to pass that in as a query parameter instead of a header? Can we support both CMR-scroll-id or scroll_id query parameter? If so let's add a test for passing in the scroll id via the CMR-scroll-id header and update the documentation.

Oh should these be dashes or underscores in :scroll-id? On the API I know it should be underscore, but I can never remember if our tests are converting them to underscores.